src/main/webapp/app/pages/LevelOfEvidencePage.tsx

```diff
@@ -188,150 +197,220 @@ export default class LevelOfEvidencePage extends React.Component<
-                  Download Slide
-                </Button>
-              )}
-              <DownloadButton
-                size={'sm'}
-                className={classnames('ml-1')}
-                href={`content/files/levelOfEvidence/${this.version}/${
-                  LEVEL_FILE_NAME[this.version]
-                }.pdf`}
-              >
-                <i className={'fa fa-cloud-download mr-1'} />
-                Download PDF
-              </DownloadButton>
-            </Col>
-          </Row>
-          <Row className={'justify-content-md-center mt-5'}>
-            <Col className={'col-md-auto text-center'}>
-              <h4>{LEVEL_TITLE[this.version]}</h4>
-              <div>
-                <span
-                  onClick={() =>
-                    this.toggleVersion(
-                      V2_RELATED_LEVELS.includes(this.version)
-                        ? Version.V1
-                        : Version.V2
-                    )
-                  }
-                  className={mainStyles.btnLinkText}
-                >
-                  {LEVEL_SUBTITLE[this.version]}
-                </span>
-              </div>
-            </Col>
-          </Row>
-          <Row>
-            <Col className={'d-md-flex justify-content-center mt-2'}>
-              <div
-                style={{
-                  maxWidth: [
-                    Version.AAC,
-                    Version.FDA,
-                    Version.V1,
-                    Version.V2,
-                  ].includes(this.version)
-                    ? undefined
-                    : IMG_MAX_WIDTH,
-                }}
-              >
-                <OptimizedImage
-                  style={{ width: '100%' }}
-                  progressiveLoading
-                  src={`content/images/level_${this.version}.png`}
-                />
-                {this.version === Version.AAC ? (
-                  <div className="text-right">
-                    <span
-                      style={{
-                        marginRight: this.props.windowStore.isLargeScreen
-                          ? '130px'
-                          : '35px',
-                      }}
-                    >
-                      <sup>1</sup>{' '}
-                      <Linkout link="https://www.sciencedirect.com/science/article/pii/S1525157816302239?via%3Dihub">
-                        Li, MM et al., J Mol Diagn 2017
-                      </Linkout>
-                    </span>
-                  </div>
-                ) : null}
-              </div>
-            </Col>
-          </Row>
-        </Tab>
-      );
-    });
+    const activeLinkStyle = {
+      fontWeight: 700,
+    };
+
+    const activeLinkUnderLine = {
+      backgroundColor: 'rgb(9,104,195)',
+      borderRadius: 20,
+      height: 4,
+      width: '50%',
+      margin: 'auto',
+      transtion: 'all 1s',
+      marginTop: 2,
+    };
 
     return (
       <DocumentTitle title={getPageTitle(PAGE_TITLE.LEVELS)}>
```

Comment:
The page title needs to be updated based on the path.

Reply:
sorry, I don't know how i managed to miss that. I'll fix it.